reporter/html.js

```diff
@@ -782,9 +782,11 @@ export function escapeText( s ) {
 					escapeText( actual ) + "</pre></td></tr>";
 
 				if ( typeof details.actual === "number" && typeof details.expected === "number" ) {
-					showDiff = true;
-					diff = ( details.actual - details.expected );
-					diff = ( diff > 0 ? "+" : "" ) + diff;
+					if ( !isNaN( details.actual ) && !isNaN( details.expected ) ) {
```

Comment:
you're right, @gibson042.

@NaridaL I'm sorry if I'm being annoying, but if you want, we could revert these changes to the original output. I appreciate your time supporting QUnit.